staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -609,3 +633,14 @@ func patchToUpdateOptions(po *metav1.PatchOptions) (*metav1.UpdateOptions, error
 	err = json.Unmarshal(b, &uo)
 	return &uo, err
 }
+
+// patchToCreateOptions creates a CreateOptions with the same field values as the provided PatchOptions.
+func patchToCreateOptions(po *metav1.PatchOptions) (*metav1.CreateOptions, error) {
```

Comment:
this implies that create options are a superset of patch options. Can we test this in a unit test and drop the error here? (or even find a way without going through JSON)

Reply:
I've removed the json conversion (https://github.com/kubernetes/kubernetes/pull/77563#discussion_r283046138), and I'm re-thinking the patch->create|update conversion, since it's lossy (patch has a "force" field that create and update do not have).